<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="APL-PG02T0CX" failures="1" tests="8" name="AutomationExcercise" time="203.307" errors="0" timestamp="2023-10-06T12:25:00 IST">
  <testcase classname="com.automation.testcases.TC1_RegistrationProcess" name="userRegistration" time="15.69"/>
  <testcase classname="com.automation.testcases.TC2_ValidateCardItems" name="validateAddItemToCartAndNonExistenceProduct" time="60.564"/>
  <testcase classname="com.automation.testcases.TC3_ValidateAddressDetailsIncheckoutPage" name="validateAddressDetailsInCheckoutPage" time="21.04"/>
  <testcase classname="com.automation.testcases.TC4_ValidateScrollUScrollDown" name="testScrollUpScrollDown" time="0.166"/>
  <testcase classname="com.automation.testcases.TC5_ValidateInvoiceAfterPurchase" name="validateInvoice" time="15.574">
    <skipped/>
  </testcase> <!-- validateInvoice -->
  <testcase classname="com.automation.testcases.TC5_ValidateInvoiceAfterPurchase" name="validateInvoice" time="0.005">
    <skipped/>
  </testcase> <!-- validateInvoice -->
  <testcase classname="com.automation.testcases.TC5_ValidateInvoiceAfterPurchase" name="validateInvoice" time="0.005">
    <skipped/>
  </testcase> <!-- validateInvoice -->
  <testcase classname="com.automation.testcases.TC5_ValidateInvoiceAfterPurchase" name="validateInvoice" time="0.005">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:87)
at org.junit.Assert.assertTrue(Assert.java:42)
at org.junit.Assert.assertTrue(Assert.java:53)
at com.automation.pageobjects.HomePage.verifyHomePage(HomePage.java:101)
at com.automation.testcases.TC5_ValidateInvoiceAfterPurchase.validateInvoice(TC5_ValidateInvoiceAfterPurchase.java:24)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- validateInvoice -->
</testsuite> <!-- AutomationExcercise -->
